components/bl-stepper-component/src/subcomponents/customized-step-title.js

```diff
@@ -0,0 +1,64 @@
+import { useMemo } from 'react'
+import { useActiveClassName } from '../helpers/use-active-class-name'
+import { StepTitleContent } from './step-title-content'
+
+export function CustomizedStepTitle(props) {
+  const { steps, currentStep, stepIndex, step, stepperClassNames } = props
```

Comment:
If we have `completed` property, does we still need currentStep? I think we could remove all logic based on current step

Reply:
if I remove currentStep, I won't be able to display the current step the user is on
![image](https://user-images.githubusercontent.com/99363993/184110757-c53e93bc-fb57-43bc-8802-b040b1465b80.png)